### PR TITLE
removing offending security libs

### PR DESCRIPTION
**Pull request by dgraeber**
*Issue #, if available:*

*Description of changes:*
Commenting out `requirements-dev.txt` entries referring to MWAA libraries due to security scan results.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by srinivasreddych — DISMISSED** [review]

**Comment by srinivasreddych** [comment]
Can you set `apache-airflow-providers-amazon` to `7.2.1` and `apache-airflow` to `2.5.2`?

File: modules/analysis/aws-batch-demo/requirements-dev.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
-apache-airflow~=2.2.4
-apache-airflow-providers-amazon~=3.3.0
+## You will need to verify the proper versions
+# of the libraries for your sue case
```
Comment (srinivasreddych) [inline modules/analysis/aws-batch-demo/requirements-dev.txt:2]: spell check `sue` to `use`. Also mention a user to check the current MWAA version and compare with the version added here.

File: modules/analysis/rosbag-image-pipeline/requirements-dev.txt [hunk 2]
```diff
@@ -1,4 +1,6 @@
-apache-airflow~=2.2.4
-apache-airflow-providers-amazon~=3.3.0
+## You will need to verify the proper versions
+# of the libraries for your sue case
```
Comment (srinivasreddych) [inline modules/analysis/rosbag-image-pipeline/requirements-dev.txt:2]: spell check `sue` to `use`. Also mention a user to check the current MWAA version and compare with the version added here.

File: modules/sensor-extraction/ros-to-parquet/requirements-dev.txt [hunk 3]
```diff
@@ -1,4 +1,6 @@
-apache-airflow~=2.2.4
-apache-airflow-providers-amazon~=3.3.0
+## You will need to verify the proper versions
+# of the libraries for your sue case
```
Comment (srinivasreddych) [inline modules/sensor-extraction/ros-to-parquet/requirements-dev.txt:2]: spell check `sue` to `use`. Also mention a user to check the current MWAA version and compare with the version added here.

File: modules/simulations/batch-managed/requirements-dev.txt [hunk 4]
```diff
@@ -1,4 +1,6 @@
-apache-airflow~=2.2.4
-apache-airflow-providers-amazon~=3.3.0
+## You will need to verify the proper versions
+# of the libraries for your sue case
```
Comment (srinivasreddych) [inline modules/simulations/batch-managed/requirements-dev.txt:2]: spell check `sue` to `use`. Also mention a user to check the current MWAA version and compare with the version added here.

File: modules/simulations/k8s-managed/requirements-dev.txt [hunk 5]
```diff
@@ -1,2 +1,4 @@
-apache-airflow~=2.2.4
-airflow-kubernetes-job-operator~=2.0.4
```
Comment (srinivasreddych) [inline modules/simulations/k8s-managed/requirements-dev.txt:2]: You have replaced `k8s-job-operator` with an incorrect one. can you find the right version for k8s?

Comment (dgraeber) [reply modules/simulations/k8s-managed/requirements-dev.txt:2]: I will put it back to where it was....

**Review by srinivasreddych — APPROVED** [review]

**Pull request merged**